src/services/identity/SitesService.ts

```diff
@@ -429,6 +434,7 @@ class SitesService {
       if (this.isGitHubCommitData(possibleCommit)) {
         return okAsync(possibleCommit)
       }
+      console.log("=======", possibleCommit)
```

Comment:
unintended console log?

Reply:
think this got removed already by [this](https://github.com/isomerpages/isomercms-backend/pull/587/commits/e83fe99607259b71315ee026b98d7c2fa83a04da) commit